node/src/actors/chain_manager/mod.rs

```diff
@@ -769,3 +840,133 @@ fn show_info_dr(data_request_pool: &DataRequestPool, block: &Block) {
         );
     }
 }
+
+#[cfg(test)]
+mod tests {
+    pub use super::*;
+    use witnet_data_structures::chain::{
+        ChainInfo, ConsensusConstants, Environment, EpochConstants, ReputationEngine,
+        ValueTransferOutput,
+    };
+    use witnet_data_structures::error::TransactionError;
+    use witnet_data_structures::transaction::{
+        CommitTransaction, MintTransaction, RevealTransaction,
+    };
+    use witnet_data_structures::vrf::VrfCtx;
+
+    fn empty_chain_manager() -> ChainManager {
+        let mut cm = ChainManager::default();
+        cm.sm_state = StateMachine::Synced;
+        cm.current_epoch = Some(0);
+        cm.epoch_constants = Some(EpochConstants {
+            checkpoint_zero_timestamp: 0,
+            checkpoints_period: 1,
+        });
+        cm.vrf_ctx = Some(VrfCtx::secp256k1().unwrap());
+        cm.chain_state.chain_info = Some(ChainInfo {
+            environment: Environment::Testnet1,
+            consensus_constants: ConsensusConstants {
```

Comment:
should we add Default trait for ConsensusConstants?

Reply:
Probably no because the values are different in testnet and mainnet. But the default values are stored in [config/src/defaults.rs](https://github.com/witnet/witnet-rust/blob/master/config/src/defaults.rs), so it should be possible to create methods like `ConsensusConstants::testnet` or `ConsensusConstants::mainnet`.